packages/gatsby-theme-carbon/src/components/FeedbackDialog/Comment.module.scss

```diff
@@ -0,0 +1,31 @@
+.commentContainer {
+  margin-top: $spacing-05;
+  position: relative;
+}
+
+.commentContainer textarea {
+  @include carbon--type-style('body-long-01');
+  resize: none;
+  margin-top: $spacing-03;
+  margin-bottom: 0;
+  width: 100%;
+  padding: $spacing-03;
+  border: none;
+  color: var(--cds-text-01);
+  background: var(--cds-ui-01);
+}
```

Comment:
also, just curious. for stuff like this, is there a reason we don't just use the Carbon Textarea component since it's already styled?

Reply:
Mainly inline theming/overrides. I didn't put in too much effort trying, I'll give it another go.